src/Docnet.Core/Docnet.Core.csproj

```diff
@@ -24,7 +24,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Content Include="runtimes/**" PackagePath="runtimes" Visible="true" />
```

Comment:
this is used in `dotnet pack` operation to wrap the native binaries when building a nuget package which is cross platform. Your solution will not work unless we decide to publish separate nugets for different runtimes.

Reply:
oh, i see... perhaps is there a condition we can add to copy all runtimes ? so at nuget pack it comes but at independent builds we take only what we need?